bot/sx-bot.el

```diff
@@ -0,0 +1,82 @@
+;;; sx-bot.el --- Functions for viewing different tabs.       -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2014  Artur Malabarba
+
+;; Author: Artur Malabarba <bruce.connor.am@gmail.com>
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;;
+
+
+;;; Code:
+
+(require 'sx-site)
+(require 'sx-tag)
+
+(setq sx-request-remaining-api-requests-message-threshold 50000)
+
+(require 'package)
+(package-initialize)
+
+(defcustom sx-bot-out-dir "./data/tags/"
+  "Directory where output tag files are saved."
+  :type 'directory
+  :group 'sx)
+
+
+;;; Printing
+(defun sx-bot-write-to-file (data)
+  "Write (cdr DATA) to file named (car DATA).
+File is savedd in `sx-bot-out-dir'."
+  (let ((file-name (expand-file-name (car data) sx-bot-out-dir)))
+    (with-temp-file file-name
+      (let* (print-length
+             (repr (prin1-to-string
```

Comment:
I was actually mistaken in my first assessment. I apparently don't know how to use `du` :) 2.6MB uncompressed, just under 600K compressed. Seems worthwhile to me, but that does mean we're tracking a compressed file (a git no-no as far as style is concerned). This means we add 600K to the repository history every time the bot runs.